src/reducetensor.cpp

```diff
@@ -62,11 +61,7 @@ namespace detail {
 
 static bool IsDynamicReductionEnabled()
```

Comment:
Please just remove this function, which is introduced in workaround PR

Reply:
Removed

Reply:
[Note] I'd rather keep it (since it costs nothing, but gives some flexibility that might be useful in the future), but okay.